src/z3c/rml/chart.py

```diff
@@ -15,8 +15,9 @@
 """
 import reportlab.lib.formatters
 from reportlab.graphics import shapes
-from reportlab.graphics.charts import barcharts, lineplots, piecharts
-from reportlab.graphics.charts import spider, doughnut
+from reportlab.graphics.charts import (barcharts, doughnut, lineplots,
+                                       piecharts, spider)
```

Comment:
I am always surprised this import format is preferred. I like the original one better.

Reply:
you can always change the `isort` configs